### PR TITLE
Description edit

### PR DESCRIPTION
**Pull request by asantewa**
50 fonts edited - in 4 commits 
- Hind Multiscript
- The Bluemlein Scripts
- Fell Types
- Batch of 25

### CODE REVIEW

File: ofl/orbitron/DESCRIPTION.en_us.html [hunk 1]
```diff
@@ -1,2 +1,9 @@
...
-<p>Orbitron was remastered as a variable font in 2019. To contribute updates or file issues, see <a href="https://github.com/theleagueof/orbitron">https://github.com/theleagueof/orbitron</a>.</p>
+<p>
+Orbitron is a geometric sans-serif typeface intended for display purposes. Orbitron was designed with the intention of giving graphic designers 
+set in the future, alternatives to typefaces like Eurostile or Bank Gothic.
```
Comment (m4rc1e) [inline ofl/orbitron/DESCRIPTION.en_us.html]: I feel this description also needs refining.

File: ofl/librecaslontext/DESCRIPTION.en_us.html [hunk 2]
```diff
@@ -1,11 +1,9 @@
...
\ No newline at end of file
+<p>
+Libre Caslon Text, a typeface based on the hand lettering artist – Caslon’s interpretations, that were typical of 1950s advertising, 
+is specifically tailored to be used for web body text (typically set at 16px).
```
Comment (m4rc1e) [inline ofl/librecaslontext/DESCRIPTION.en_us.html]: I think we can drop "(typically set at 16px)"

**Comment by asantewa** [comment]
I consent

**Comment by tphinney** [comment]
That is a messed up sentence: “Libre Caslon Text, a typeface based on the hand lettering artist – Caslon’s interpretations, that were typical of 1950s advertising.”

A sentence that works would be: “Libre Caslon, instead, is based on hand lettering artist interpretations of Caslon designs, typical of 1950s advertising.

**Pull request closed without merging**